uaclient/cli.py

```diff
@@ -837,7 +832,7 @@ def get_parser():
         help="refresh Ubuntu Advantage services from contracts server",
     )
     parser_fix = subparsers.add_parser(
-        "fix"  # BETA omit help string to hide subcommand from help
+        "fix", help="fix CVE and USN security issues on this machine"
```

Comment:
what about 

    help="check for and mitigate the impact of a CVE/USN on this system"